acme_diags/derivations/acme.py

```diff
@@ -348,11 +362,11 @@ def cosp_bin_sum(cld, prs_low0, prs_high0, tau_low0, tau_high0):  # noqa
                 cosp_prs=(prs_low, prs_high),
                 cosp_tau_modis=(tau_low, tau_high),
             )  # MODIS model
-            if prs_high == 440:
+            if prs_low in [440, 44000]:
                 prs_lim = "high cloud fraction"
-            if prs_high == 680 and prs_low == 440:
+            if prs_low in [680, 68000] and prs_high in [440, 44000]:
```

Comment:
Is this line supposed to be `if prs_low in [440, 44000] and prs_high in [680, 68000]`?

Reply:
prs are cloud top **height** (increase in height: i.e. 0, 250, 750, 1250, 1750, 2250, 2750, 3500, 4500, 6000, 8000, 10000, 12000, 14500, 16000, 18000 in units meter) or cloud top **pressure** (decrease in height: 90000, 74000, 62000, 50000, 37500, 24500, 9000 in units Pa). So the bounds are correct here...